Server/src/main/java/local/AssetDAOImpl.java

```diff
@@ -232,6 +232,17 @@ public void insertAsset(Asset asset) {
         }
     }
 
+    public void updateRecommendation(int assetID, String recommendation) {
```

Comment:
Add the @override tag so this method is shown in the interface that this class extends.

Reply:
Resolved in the last commit.